include/queue.php

```diff
@@ -29,14 +29,6 @@ function queue_run(&$argv, &$argc){
 		// Handling the pubsubhubbub requests
 		proc_run(PRIORITY_HIGH,'include/pubsubpublish.php');
 
-		$r = q("select * from deliverq where 1");
```

Comment:
See above. The table "deliverq" is not used anymore. And maybe in future I will replace the "queue" table as well.